core/controllers/admin.py

```diff
@@ -1066,3 +1066,30 @@ def get(self):
             'number_of_pending_deletion_models': (
                 wipeout_service.get_number_of_pending_deletion_requests())
         })
+
+
+class GetModelsRelatedToUserHandler(base.BaseHandler):
```

Comment:
FYI this is no longer needed for Brian's thing, but not sure if you still want to keep it around. If so I think that's OK too.